packages/design-system/src/components/primitives/numeric-gesture-control.ts

```diff
@@ -14,21 +14,23 @@
  * });
  */
 
-export type Direction = "horizontal" | "vertical";
+export type NumericGestureDirection = "horizontal" | "vertical";
 
-export type Value = number;
+export type NumericGestureValue = number;
```

Comment:
Why not just a NumericValue ?

Reply:
btw we discovered recently a better name for that thing: "scrub", its an actual thing for this interaction, so this component could be NumericScrubControl

Reply:
or simply ScrubControl if its always numeric

Reply:
same for the rest of the related primitives